spec/mailers/previews/company_mailer_preview.rb

```diff
@@ -1,11 +1,11 @@
 class CompanyMailerPreview < ActionMailer::Preview
-  def decline_email
-    quote = Quote.last
-    CompanyMailer.decline_email(quote)
+  def decline_email_preview
+    # quote = Quote.last
+    CompanyMailer.decline_email(Quote.last)
```

Comment:
Which controller? This is the mailer preview, so would it be the actual CompanyMailer?
```class CompanyMailer < ApplicationMailer
  
  def decline_email(quote)
    @quote = quote
    @company = quote.company

    mail to: @company.email,
         subject: 'Your quote was rejected by the customer.'
    mail to: @company.email,
         subject: "Your quote was rejected by the customer"
  end